prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -8,18 +8,41 @@
 
 namespace Prime.Services
 {
+    public class EmailProps
```

Comment:
Props is a rather Javascript term :P
Razor wants to render a _model_ even though we are really just smushing arguments into an object. How about something like EmailBody?

Reply:
EmailBody suggests structure and these are just parameters for binding.  What about EmailParams?

Reply:
I've updated it to be EmailParams